.vscode/settings.json

```diff
@@ -1 +1,2 @@
-{}
\ No newline at end of file
+{
+}
```

Comment:
Not sure if this change matters? :thinking:  Probably not though..

Reply:
Yeah just an annoying windows thing. I will revert it before merging